tests/FakeItEasy.Specs/CallMatchingSpecs.cs

```diff
@@ -479,27 +479,24 @@ public interface IHaveANullableParameter
 
         public static IEnumerable<object[]> Fakes()
         {
-            yield return new object[] { A.Fake<object>(), "Faked " + typeof(object).FullName };
-            yield return new object[] { A.Fake<object>(), "Faked " + typeof(object).ToString() };
-            yield return new object[] { A.Fake<List<int>>(), "Faked " + typeof(List<int>).FullName };
-            yield return new object[] { A.Fake<IList<int>>(), "Faked " + typeof(IList<int>).FullName };
+            yield return new object[] { A.Fake<object>(), "Faked " + typeof(object) };
+            yield return new object[] { A.Fake<List<int>>(), "Faked " + typeof(List<int>) };
+            yield return new object[] { A.Fake<IList<int>>(), "Faked " + typeof(IList<int>) };
             yield return new object[] { A.Fake<Action<int>>(), typeof(Action<int>).ToString() };
         }
 
         public static IEnumerable<object[]> FakesWithBadToString()
         {
-            yield return new object[] { A.Fake<object>(), "Faked " + typeof(object).FullName };
-            yield return new object[] { A.Fake<object>(), "Faked " + typeof(object).ToString() };
-            yield return new object[] { A.Fake<List<int>>(), "Faked " + typeof(List<int>).FullName };
-            yield return new object[] { A.Fake<IList<int>>(), "Faked " + typeof(IList<int>).FullName };
+            yield return new object[] { A.Fake<object>(), "Faked " + typeof(object) };
+            yield return new object[] { A.Fake<List<int>>(), "Faked " + typeof(List<int>) };
+            yield return new object[] { A.Fake<IList<int>>(), "Faked " + typeof(IList<int>) };
         }
 
         public static IEnumerable<object[]> StrictFakes()
         {
-            yield return new object[] { new Func<object>(() => A.Fake<object>(o => o.Strict())), "Faked " + typeof(object).FullName };
-            yield return new object[] { new Func<object>(() => A.Fake<object>(o => o.Strict())), "Faked " + typeof(object).ToString() };
-            yield return new object[] { new Func<object>(() => A.Fake<List<int>>(o => o.Strict())), "Faked " + typeof(List<int>).FullName };
-            yield return new object[] { new Func<object>(() => A.Fake<IList<int>>(o => o.Strict())), "Faked " + typeof(IList<int>).FullName };
+            yield return new object[] { new Func<object>(() => A.Fake<object>(o => o.Strict())), "Faked " + typeof(object) };
+            yield return new object[] { new Func<object>(() => A.Fake<List<int>>(o => o.Strict())), "Faked " + typeof(List<int>) };
+            yield return new object[] { new Func<object>(() => A.Fake<IList<int>>(o => o.Strict())), "Faked " + typeof(IList<int>) };
             yield return new object[] { new Func<object>(() => A.Fake<Action<int>>(o => o.Strict())), typeof(Action<int>).ToString() };

```

Comment:
Nope. It will be passed as the fake description to `PassingAStrictFakeToAMethod`, so it needs to be a string, not a type. I omitted `ToString()` on the other lines, because it's called implicitly for the concatenation with "Faked ".